app/models/rss_feed.rb

```diff
@@ -1,6 +1,7 @@
 require 'resource_feeder/common'
 class RssFeed < ApplicationRecord
   include ResourceFeeder
+  include_concern 'ImportExport'
```

Comment:
Thanks @lpichler.  I guess I didn't even expect that this module was already included... I moved the include with the other include so it's more obvious.

👍